interface/src/app/analyze/analysis/analysis.js

```diff
@@ -163,11 +160,28 @@ AnnotationType, SampleBin) {
           'enter': {
             'x': {'scale': 'nodes', 'field': 'node'},
             'width': {'scale': 'nodes', 'band': true},
-            'y': {'scale': 'samples', 'field': 'sample'},
+            'y': {'scale': 'samples', 'field': 'sample_order._id'},
             'height': {'scale': 'samples', 'band': true},
             'fill': {'scale': 's', 'field': 'normval'}
           }
         }
+      }, {
+        'type': 'text',
+        'from': {'data': 'samples'},
+        'properties': {
+          'update': {
+            'text': {'field': 'data'},
+            'x': {'value': -5},
+            'y': {
+              'scale': 'samples',
+              'field': '_id',
+              'offset': 18
```

Comment:
'offset' here and 'value' above in line 174 seem a little bit like magic numbers here. Where are they coming from and how do we decide on their values?